gce/api-servers.tf

```diff
@@ -1,7 +1,7 @@
 /* API server */
 resource "google_compute_instance" "api" {
   count = 2
-  name = "tsuru-api-${count.index}"
+  name = "tsuru-api-${var.env}-${count.index}"
```

Comment:
Nitpick, I think this should be the other way round. So that we know the environment is always the suffix.